apps/aesophia/test/contracts/complex_types.aes

```diff
@@ -78,8 +78,8 @@ contract ComplexTypes =
       None :: ys => None
       Some(x) :: ys =>
         switch(all_some(ys))
-          None     => None
           Some(xs) => Some(x :: xs)
+          None     => None
```

Comment:
I understand this was the symptom.

It may be worth a release notes line so that users experiencing same symptom realize that they shall retry on latest release (once this fix is included in a release).

Reply:
Did we change the way we handle release notes again? Should I add a file to `docs/release-notes/next/`?